tox.ini

```diff
@@ -42,16 +42,22 @@ commands =
     test: coverage html --directory=bin/htmlcov
     clean: make clean {posargs}
     codegen: make {posargs}
-    flake8: flake8 {posargs}
+    flake8: flake8 bin/ 
+    flake8: flake8 src/nidmm/system_tests/ 
```

Comment:
what about the unit tests, are those being flake8ed?

Reply:
Yes, they are still copied into bin for the 'flake8 bin/' command will test them.

Reply:
Do they have to be copied, or can they be run from src/?

Reply:
They need to be copied anyway for pytest because we are not installing anything. We rely on the relative positions in bin for the driver module to load correctly. 

For flake8 they can be tested in src.

Reply:
We could install, however, since we run inside a virtual environment, right?
I ask so I can understand, not because we necessarily need to change things.